inspirehep/modules/records/receivers.py

```diff
@@ -179,8 +192,7 @@ def index_after_commit(sender, changes):
             index_modified_citations_from_record.delay(pid_type, pid_value, db_version)
 
 
-@before_record_index.connect
-def enhance_before_index(sender, json, record, *args, **kwargs):
+def enhance_before_index(sender, record, *args, **kwargs):
```

Comment:
since we removed the decorator ``@before_record_index.connect`` why we keep the same signature? do we need the ``sender``?